<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vocabulary</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/img/ivocicon.png</normaloff>:/icons/img/ivocicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Keresés:</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_searchfor</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_searchfor"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_search">
          <property name="text">
           <string>Keresés</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 14pt &quot;Nimbus Roman No9 L&quot;;</string>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionRectVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_searchResults">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_word"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_meaning"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addModify">
          <property name="text">
           <string>Felvesz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReload"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEleemk">
    <property name="title">
     <string>Szavak</string>
    </property>
    <addaction name="actionGo_To_Words"/>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Keresés</string>
    </property>
    <addaction name="actionSearch_For"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Súgó</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEleemk"/>
   <addaction name="menuSearch"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionGo_To_Words"/>
   <addaction name="actionSearch_For"/>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Mentés</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_j">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>Új</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Szerkesztés</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionGo_To_Words">
   <property name="text">
    <string>Ugrás szavakhoz</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Törlés...</string>
   </property>
   <property name="toolTip">
    <string>Törlés...</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSearch_For">
   <property name="text">
    <string>Ugrás Kereséshez</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Újra betöltés</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Névjegy...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>lineEdit_word</tabstop>
  <tabstop>lineEdit_meaning</tabstop>
  <tabstop>pushButton_addModify</tabstop>
  <tabstop>lineEdit_searchfor</tabstop>
  <tabstop>pushButton_search</tabstop>
  <tabstop>listWidget_searchResults</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
